dev_guide/projects.adoc

```diff
@@ -104,8 +110,16 @@ Selecting a project brings you to the
 xref:../architecture/infrastructure_components/web_console.adoc#project-overviews[project
 overview] for that project.
 
-[[check-project-status]]
+Clicking on the kebab menu for a particular project presents you with the
+following options:
 
+image::3-7-project-kebab-options.png[Project Kebab Menu]
+
+The right panel shown with the Service Catalog provides quick access to the most
```

Comment:
@ahardin-rh I'm wondering if it makes sense to state this before showing the full projects list because upon login, the user will no longer land on the full projects page, they will be presented with the new landing page (including this projects panel). For users with more than 5 projects they can click "View All" to navigate to the full projects list (the screenshot you show above).